src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -242,6 +242,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1503 = 1503,
 
+        /// <summary>
+        /// No metadata file found
+        /// </summary>
+        NU1504 = 1504,
```

Comment:
This isn't really an input error. I think an 1800 or 1900 might be better. 
Leaning toward 1900 myself, but would love to hear what @zivkan thinks.

Reply:
I guess 1800 makes more sense. 
Let's do 1800 instead.

Reply:
Ok, done.